tests/func-tests/node_placement_test.go

```diff
@@ -95,19 +177,21 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
 				"virt-controller": false,
 			}
 
-			By("Listing infra nodes")
-			infraNodes := listInfraNodes(client)
+			Eventually(func(g Gomega) {
```

Comment:
Here too - for functions used here - if they are using gomega - use them from the object.